.circleci/config.yml

```diff
@@ -128,6 +132,7 @@ aliases:
         type: boolean
         default: false
     environment:
+      <<: *e2e-executor-env
       CYPRESS_PROJECT_ID: s3j3qj
       CYPRESS_RECORD_KEY: 3904ca0c-bc98-47d9-8371-b68c5e81fb9b
```

Comment:
Just example of changes I applied across this file - in short we need to define "base" env vars otherwise they are lost (they are rewritten by new ones and not merged in by default)